SAM2_Tracking/utils.py

```diff
@@ -98,7 +100,7 @@ def adjust_annotations(annotations_file=None, fps=None, SAM2_start=None,
     df = df[df_columns]
 
     # Correct annotation frame value, so it coincides with video frame value
-    df[frame_col_name] = (df[frame_col_name] - SAM2_start) / (fps / 3)
+    df[frame_col_name] = (df[frame_col_name] / (fps / out_fps) + SAM2_start)
```

Comment:
```suggestion
    df[frame_col_name] = df[frame_col_name] * (fps / out_fps) + SAM2_start
```
The formula you put above does not match this calculation. If the formula you put above is correct, then I think you want my change.